test/mrp/docker-compose.yml

```diff
@@ -0,0 +1,83 @@
+services:
+  zookeeper:
+    image: wurstmeister/zookeeper
+    container_name: zookeeper
+    ports:
+       - "2181:2182"
+    volumes:
+      - /etc/localtime:/etc/localtime:ro
+      - /etc/timezone:/etc/timezone:ro
+      - zookeeper-datavolume:/opt/zookeeper-3.4.13/data
```

Comment:
I believe this will fail since we have not define this volume. Please either copy the volume section of the carma-streets docker compose relevant for this or remove this volume from the service.

Reply:
Two possible solution:
1. I will copy the necessary volume from the carma-streets repo regarding starting kafka and zookeeper etc.
2. I will delete the docker-compose file here and then link the original carma-street repo and let the uses run "''' docker compose" under original carma street repo. 
What is your suggestions?